app/templates/index.hbs

```diff
@@ -0,0 +1,12 @@
+
+<h1>remEMBER</h1>
+
+{{#each model as |reminder|}}
```

Comment:
So if this is a list of reminders, I would expect that to be in a reminders template.